media/css/firefox/features/common.scss

```diff
@@ -212,18 +215,15 @@ main {
 /* -------------------------------------------------------------------------- */
 // Download button overrides
 
-.fx-privacy-link {
-    a:link,
-    a:visited {
-        @include font-size(14px);
-        @include open-sans();
-        color: $color-burnt-orange;
-        text-decoration: none;
-    }
+.fx-privacy-link a:link,
+.fx-privacy-link a:visited {
+    @include font-size(12px);
+    color: $color-link-blue;
+    font-family: $font-open-sans;
+    text-decoration: none;
 
-    a:hover,
-    a:active,
-    a:focus {
-        text-decoration: underline;
+    &:hover,
+    &:focus {
+        color: $color-link-blue-dark;
```

Comment:
It would be nice not to lose the underline on hover here? (same for other occurrences)